sync/class.jetpack-sync-actions.php

```diff
@@ -439,7 +439,7 @@ static function cleanup_on_upgrade( $new_version = null, $old_version = null ) {
 		}
 	}
 
-	static function get_sync_status() {
+	static function get_sync_status( $fields = null ) {
```

Comment:
It would be great if fields was always an array. instead of sometimes an array and sometimes a string.

Reply:
This is a helper function. Query params need to be strings anyway and I think this is the most straightforward way to handle things.

Reply:
We could also support both `array` and `string` (like many other WP core functions and methods do), but we should make sure that's properly supported by adding specific tests for that.

Reply:
I'd say we don't need that because this is just a helper function used in a single JSON API response.

Reply:
So we never really using anything but a string.